samples/tab-stage-view/nodejs/server/models/adaptiveCardUnfurling.js

```diff
@@ -0,0 +1,41 @@
+// Adaptive Card for tab to show in stage view
```

Comment:
This adaptive care you are showing in tab or as a card for link unfurling?

Reply:
Its card as link unfurling which opens a tab in stage view